### PR TITLE
Move oak_remote_attestation/proto to proto/session

### PR DESCRIPTION
**Pull request by k-naliuka**
Ref https://github.com/project-oak/oak/issues/4392

### CODE REVIEW

**Comment by tiziano88** [comment]
Did we choose to go with `session` instead of `attestation`?

**Comment by ipetr0v** [comment]
The reason for the `session` name is because we'll eventually remove all attestation related logic from these protos and will just use DICE attestation provided by https://github.com/project-oak/oak/tree/main/proto/attestation

**Review by tiziano88 — APPROVED** [review]
LG but I'd like to see a more detailed plan of what's staying in those protos and what will be deleted, and where that is tracked

File: proto/session/v1/service_streaming.proto [hunk 1]
```diff
@@ -18,7 +18,7 @@ syntax = "proto3";
 
 package oak.session.v1;
 
-import "oak_remote_attestation/proto/v1/messages.proto";
+import "proto/session/v1/messages.proto";
```
Comment (ipetr0v) [inline proto/session/v1/service_streaming.proto:21]: I think we are no longer using `v1` in the directory names in the new `proto` direcroty. `v1` is only mentioned in the package names.

So the directory could just be `proto/session/file.proto`

**Review by ipetr0v — APPROVED** [review]
Thanks!

**Comment by ipetr0v** [comment]
> LG but I'd like to see a more detailed plan of what's staying in those protos and what will be deleted, and where that is tracked

Created a new issue: https://github.com/project-oak/oak/issues/4627

File: proto/session/v1/service_streaming.proto (hunk 1 above)
Comment (k-naliuka) [reply proto/session/v1/service_streaming.proto]: Removed the v1 subdir but left v1 in namespaces - please let me know if you'd prefer them to go too!

**Review by ipetr0v — APPROVED** [review]

**Pull request merged**